recipes/nodejs/all/conanfile.py

```diff
@@ -1,74 +1,66 @@
 import os
-import re
-from six import StringIO
-from conan import ConanFile, conan_version
-from conan.tools.scm import Version
-from conan.tools.files import copy, get
+
+from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import copy, get
 
 required_conan_version = ">=1.59.0"
 
 
 class NodejsConan(ConanFile):
     name = "nodejs"
-    description = "nodejs binaries for use in recipes"
-    topics = ("node", "javascript", "runtime")
+    description = "Node.js is an open-source, cross-platform JavaScript runtime environment."
+    license = "MIT"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://nodejs.org"
-    license = "MIT"
+    topics = ("node", "javascript", "runtime", "pre-built")
+
     package_type = "application"
-    settings = "os", "arch", "compiler"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
     short_paths = True
 
-    @property
-    def _source_subfolder(self):
-        return os.path.join(self.source_folder, "source_subfolder")
+    def layout(self):
+        pass
```

Comment:
Why do you need this no-op definition of `layout()`?

Reply:
To disable a linter warning / convention.